airflow/www/views.py

```diff
@@ -5803,21 +5853,42 @@ def duration_f(self):
     @provide_session
     @action_logging
     def action_clear(self, task_instances, session: Session = NEW_SESSION):
-        """Clears the action."""
+        """Clears an arbitrary number of task instances."""
         try:
-            dag_to_tis = collections.defaultdict(list)
-
-            for ti in task_instances:
-                dag = get_airflow_app().dag_bag.get_dag(ti.dag_id)
-                dag_to_tis[dag].append(ti)
+            count = self._clear_task_instances(
+                task_instances=task_instances, session=session, clear_downstream=False
+            )
+            session.commit()
+            flash(f"{count} task instances have been cleared")
```

Comment:
Thoughts on something like:
```suggestion
            flash(f"{count} task instance{'s have' if count > 1 else ' has'} been cleared")
```

In order to read as `1 task instance has been cleared` / `4 task instances have been cleared`?

Reply:
Sure, good idea!